jenkins/dockerfiles/build-and-push.sh

```diff
@@ -121,3 +122,15 @@ then
     docker push $tag
   fi
 fi
+
+if [ "x${fastboot_docker}" == "xtrue" ]
+then
+  tag=${tag_px}fastboot_docker
```

Comment:
```suggestion
  tag=${tag_px}fastboot
```
It's obviously a Docker image, no need to add `docker` in the name.

Reply:
(same thing in other places)